tools/generator/dfg/rp/rp_groups.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020 Hannes Ellinger
+# All rights reserved.
+
+rp_groups = \
+[
+    # RP devices
+    {
+        'name': ['2040']
```

Comment:
```suggestion
        'family': ['20']
```
with the change in the DID.